src/views/VaultDetails/components/helpers/Stable3PoolDeposit.tsx

```diff
@@ -19,25 +30,40 @@ import {
 import BigNumber from 'bignumber.js'
 
 const { Title, Text } = Typography
-const { useBreakpoint } = Grid
-
-const Currencies3Pool = [DAI, USDT, USDC]
-
-const initialCurrencyValues: CurrencyValues = reduce(
-	Currencies3Pool,
-	(prev, curr) => ({
-		...prev,
-		[curr.tokenId]: '',
-	}),
-	{},
-)
+// const { useBreakpoint } = Grid
 
 /**
  * Creates a deposit table for the savings account.
  */
-export default function Stable3PoolDeposit({ set3crvValue, value3crv }) {
+export default function Stable3PoolDeposit({ set3crvValue, value3crv, vault }) {
+	const [Currencies3Pool, setCurrencies3Pool] = useState<any>([])
+
+	useEffect(() => {
```

Comment:
Every time we add a new Vault this deeply nested component will need to be modified.

What if instead we held the names of the tokens that make up a Curve LP on the static configuration for them?
https://github.com/yaxis-project/yaxis-frontend-v2/blob/master/src/constants/configs/ethereum/mainnet.ts#L38